nemo_skills/dataset/speed-bench/prepare.py

```
@@ -0,0 +1,635 @@
+    for i, answer in enumerate(answers):
+        if i == correct_answer_i:
+            continue
+        answer_to_add = f"A{i + 1}:\n\n{answer}\n\n"
+        answer_to_add_tokens = len(encoder.encode(answer_to_add, disallowed_special=()))
+        if all_tokens + answer_to_add_tokens > num_tokens:
+            break
+        answers_to_add_stop = i
+    answers_to_add = (
+        answers[: answers_to_add_stop + 1]
+        if answers_to_add_stop >= correct_answer_i
+        else [answers[correct_answer_i]] + answers[: answers_to_add_stop + 1]
```

Ruff (0.15.2)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)